README.md

```diff
@@ -42,233 +42,88 @@ require 'path/to/anet_php_sdk/autoload.php';
 To authenticate with the Authorize.Net API you will need to retrieve your API Login ID and Transaction Key from the [`Merchant Interface`](https://account.authorize.net/).  You can find these details in the Settings section.
 If you need a sandbox account you can sign up for one really easily [`here`](https://developer.authorize.net/sandbox/).
 
-Once you have your keys simply plug them into the appropriate variables as per the samples below.
+Once you have your keys simply plug them into the appropriate variables, as per the below code dealing with the authentication part of the flow.
 
 ````php
-define("AUTHORIZENET_API_LOGIN_ID", "YOURLOGIN");
-define("AUTHORIZENET_TRANSACTION_KEY", "YOURKEY");
+$merchantAuthentication = new AnetAPI\MerchantAuthenticationType();
+$merchantAuthentication->setName("YOURLOGIN");
+$merchantAuthentication->setTransactionKey("YOURKEY");
 ````
+...
+
+````php
+$request = new AnetAPI\CreateTransactionRequest();
+$request->setMerchantAuthentication($merchantAuthentication);
+````
+...
 
 ## Usage Examples
+*Users of previous SDK model please note: The SDK is moving towards using a new model. This model allows us to maintain SDKs better by being more responsive to API changes. (To determine if a code is using the old model or new, a prominent difference is the usage of controllers, which is used only with the new model.) Refer the [old README](README_OLD.md) for samples of the old model.*
 
```

Comment:
I don't think we need to refer to old model in the updated readme.  We should keep the old usage examples but just put a readme in the top of the old model folders and clearly say this is deprecated funtionality.